common.css

```diff
@@ -86,6 +89,27 @@ span{
     display: grid;
     grid-template-columns: 300px auto;
 }
+.quiz-heading-container{
+    margin: 2rem 2rem;
+}
+.quiz-btn-container{
+    margin: 2rem 2rem;
+}
+.quiz-info-container{
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    max-width: 1200px;
```

Comment:
You can use rem instead of px here too.

Reply:
Here i dont want to change its width according to font size

Reply:
> Here i dont want to change its width according to font size

Good to know. Rest looks good to merge.